boudicca.base/common-model/src/jvmTest/kotlin/base/boudicca/model/EventEntryConversionTest.kt

```diff
@@ -34,9 +37,10 @@ class EventEntryConversionTest {
             "description:format=markdown:lang=en" to "#my english markdown description"
         )
 
-        val event = Event.fromEntry(entry).get()
+        val event = Event.fromEntry(entry)
 
-        assertEquals(6, event.data.size)
+        assertThat(event).isNotNull()
+        assertEquals(6, event!!.data.size)
```

Comment:
before merging replace assertEquals

Reply:
again, changing out assertj with assertk has nothing to do with this PR, so I will do it in another PR

Reply:
but now we use assertk in oneline and just for one another line we have a different assertion. it would be good to stay consistent. it is not that many lines we would need to change. and it would keep up with the spirit of fixing things as we touch them.